### PR TITLE
Parse dependency versions

### PR DESCRIPTION
**Pull request by jeroen**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,7 +59,15 @@ function parse_desc_stream(descstream, callback) {
...
-  return str.split(/,[\s]*/).map(normalize_ws);
+  return str.split(/,[\s]*/).map(function(str){
+    return str.match(/\(.+\)/) ?
+        {
+            package: str.replace(/\(.+\)/, '').trim(),
+            version: str.replace(/.*\((.+)\)/, '$1').trim()
+        } : {
+            package: str.trim()
+        };
+    });
```
Comment (gaborcsardi) [inline index.js:76]: Does this handle the case where there are newlines within the dependency, e.g. `dplyr\n(>= 0.5.0)` or `dplyr (>= \n0.5.0)`?

Comment (jeroen) [reply index.js:76]: Good question :) I'll add a test case and see.

**Comment by jeroen** [comment]
Thanks for the feedback! Added some more corner cases to the tests, and squashed my commits.

**Comment by gaborcsardi** [comment]
Thanks!
